as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/SmokeTestsSuite.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.tools.as.ui.bot.itests;
```

Comment:
Please add Copyright notice to the all java files as a header. see https://github.com/jbosstools/jbosstools-server/blob/master/as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/parametized/server/ServerRuntimesTest.java#L1